prov/efa/src/rxr/rxr.h

```diff
@@ -467,6 +467,43 @@ struct rxr_tx_entry {
 #endif
 };
 
+static inline
+size_t rxr_copy_from_tx(void *buf, size_t tocopy,
+			struct rxr_tx_entry *tx_entry, size_t offset)
+{
+	size_t data_size;
+
+#ifdef HAVE_LIBCUDA
+	if (rxr_ep_is_cuda_mr(tx_entry->desc[0]))
+		data_size = ofi_copy_from_cuda_iov(buf,
+						   tocopy,
+						   tx_entry->iov,
+						   tx_entry->iov_count,
+						   offset);
+       else
+#endif
+		data_size = ofi_copy_from_iov(buf,
+					      tocopy,
+					      tx_entry->iov,
+					      tx_entry->iov_count,
+					      offset);
+	return data_size;
+}
+
+static inline
+size_t rxr_copy_to_rx(void *data, size_t tocopy, struct rxr_rx_entry *rx_entry, size_t offset)
+{
+	size_t data_size;
+#ifdef HAVE_LIBCUDA
+	if (rxr_ep_is_cuda_mr(rx_entry->desc[0]))
+		data_size = ofi_copy_to_cuda_iov(rx_entry->iov, rx_entry->iov_count, offset, data, tocopy);
```

Comment:
Can you format this function the same as the tx one?

Reply:
Sure.